bin/codeclimate-rubocop

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env ruby
-$LOAD_PATH.unshift(
+$:.unshift(
```

Comment:
Yep, this was recommended by RuboCop in this [rule](http://www.rubydoc.info/github/bbatsov/rubocop/Rubocop/Cop/Style/SpecialGlobalVars).